sycl/doc/extensions/proposed/sycl_ext_oneapi_graph_fusion.asciidoc

```diff
@@ -200,44 +260,34 @@ this buffer will inherit this property (unless overridden).
 `sycl::aligned_alloc_shared()` to apply the property to an USM pointer.
 
 ```c++ 
-sycl::ext::oneapi::experimental::property::promote_private 
+sycl::ext::oneapi::experimental::property::fusion_internal_memory 
```

Comment:
I'd prefer if we could come up with a name that isn't tied to fusion.  But I'm happy to defer that until later.

Reply:
We originally considered to name this just `internal_memory`, but were unsure whether this would cause future collisions/confusion if other extensions wanted to introduce a similar property, but with different semantics.

I'm happy to rename it, if you prefer. Alternatively, we could change the prefix from `fusion` to `graph` or add another namespace, e.g, `...::property::graph::internal_memory` or `...::property::fusion::internal_memory`. 

Which variant would you prefer?

Reply:
You've convinced me that it's better to be explicit for now.

I think what I ultimately want is a way to describe that the results of some memory can be discarded, that isn't tied to graphs _or_ fusion.  But we should get some implementation experience with what you've spec'd here, and keep an eye out for other extensions looking for similar properties.  You're right that we don't want things to be confusing here, and so we should wait to introduce the broad thing until we're more confident it's a good idea.